drafthorse/models/elements.py

```diff
@@ -368,6 +368,56 @@ def __str__(self):
         return "{}".format(self._value)
 
 
+class FormattedDateTimeElement(StringElement):
+    def __init__(self, namespace, tag, value=None, format="102"):
+        super().__init__(namespace, tag)
+        self._value = value
+        self._format = format
```

Comment:
This class looks good, but it is the kind of code that would profit from a few extra tests to make sure the format conversions are correct back and forth. would you mind adding those?